app/src/main/java/camp/nextstep/support/jdbc/init/DatabasePopulatorUtils.java

```diff
@@ -42,5 +42,32 @@ public static void execute(final DataSource dataSource) {
         }
     }
 
+    public static void cleanup(final DataSource dataSource) {
+        Connection connection = null;
+        Statement statement = null;
+        try {
```

Comment:
```suggestion
        try (
                final Connection connection = dataSource.getConnection();
                final Statement statement = connection.createStatement()
        ) {
```

혹시 try-with-resources 문법을 활용하지 않은 이유가 있으시나요?
이미 알고 계셨을 것 같은데 일부러 null 로 초기화를 한 듯 해서 궁금해서 여쭈어봅니다.

Reply:
아하, 사실 이 부분은 이미 있는 소스코드를 그냥 썼는데, 너무 무신경하게 사용했군요. 😅  깔끔하게 다듬어보겠슴다!

Reply:
반영 완료! [311995b](https://github.com/next-step/java-jdbc/pull/16/commits/311995b2aeba4fb32e7d049d3e9339fc468ee078)